datareservoirio/client.py

```diff
@@ -337,6 +337,7 @@ def wrapper(self, series_id, start=None, end=None, **kwargs):
         wait=wait_chain(*[wait_fixed(0.1), wait_fixed(0.5), wait_fixed(30)]),
         before_sleep=before_sleep_log(log, logging.INFO),
     )
+    @log_retry
```

Comment:
I am guessing, but this will log all exceptions, such ValueError? It is the retry filter that ignores exceptions of a certain type... I am not sure, I guess that is fine, but it is not nescessarily a retry that is logged. I think it is fine that we log all exceptions as warnings, but I am not so certain we are logging only the network exceptions and things that are retried.

Lets say we get a ValueError, I assume it will be logged as a warning and we log it with
```                    "message": "exception is retried",```, but I am not so sure that is true.